server/graphql/v1/mutations/collectives.js

```diff
@@ -911,3 +915,54 @@ export async function deleteUserCollective(_, args, req) {
     })
     .then(() => userCollective);
 }
+
+export async function sendMessageToCollective(_, args, req) {
+  if (!req.remoteUser) {
+    throw new errors.Unauthorized({
+      message: 'You need to be logged in to contact a collective',
+    });
+  }
+
+  const collective = await models.Collective.findByPk(args.collectiveId);
+  if (!collective) {
+    throw new errors.NotFound({
+      message: `Collective with id ${args.id} not found`,
+    });
+  }
+
+  if (!(await collective.canContact())) {
+    throw new errors.Unauthorized({
+      message: `You can't contact this type of collective`,
+    });
+  }
+
+  if (!collective.isActive) {
+    throw new errors.Unauthorized({
+      message: `Collective is inactive, unable to contact.`,
+    });
+  }
+
+  const user = req.remoteUser;
+  // Limit email sent to 5 per hour
+  const maxEmailMessagePerHour = config.limits.collectiveEmailMessagePerHour;
+  const countCacheKey = `user_contact_collective_email_${user.id}`;
+  const existingCount = (await cache.get(countCacheKey)) || 0;
+  if (existingCount > maxEmailMessagePerHour) {
+    throw new errors.RateLimitExceeded();
+  }
+  const fromCollective = await models.Collective.findByPk(req.remoteUser.CollectiveId);
+  const data = {
+    fromCollective,
+    collective,
+    subject: args.subject || null,
+    message: sanitize(args.message.trim(), {
+      allowedTags: [],
+      allowedAttributes: {},
+    }),
+    user,
+  };
+  const recipient = `hello@${collective.slug}.opencollective.com`;
```

Comment:
I missed this during my first review, the goal being to remove our reliance on `hello@___` email for this feature, we should avoid using it. If that doesn't already exist, we should create a function to easily target collective admins when we send emails.